src/zkevm_specs/evm/table.py

```diff
@@ -130,7 +130,9 @@ class TxContextFieldTag(IntEnum):
     Value = auto()
     CallDataLength = auto()
     CallDataGasCost = auto()
+    TxSignHash = auto()
```

Comment:
should we sync tables.md for these changes?

Reply:
Thanks for catching this! I've updated the `tables.md` in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/243/commits/27b969d0c735242fc726134aedd4c6441584c560!